index.js

```diff
@@ -0,0 +1,95 @@
+const express = require('express');
+const knex = require('knex');
+
+const knexConfig = require('./knexfile.js');
+
+const server = express();
+
+server.use(express.json());
+
+// connect to the database
+const db = knex(knexConfig.development);
+
+server.get('/', (req, res) => {
+  res.send('api working');
+});
+
+
+server.get('/api/cohorts', (req, res) => {
+  db('cohorts')
+    .then(cohorts => {
+      res.status(200).json(cohorts);
+    })
+    .catch(err => res.status(500).json(err));
```

Comment:
There's a lot of repetition for your callbacks throughout. Try to see places where you might be able to break out some of the code into helper functions. For example:
```javascript
const sendOK = res => data => {
  res.status(200).json(data);
};

const sendErr = res => err => {
  res.status(500).json(err);
};
```
and usage:
```javascript
db('cohorts')
  .then(sendOK(res))
  .catch(sendErr(res));
```